docs/source/guides/libraries/swift_ios.rst

```diff
@@ -0,0 +1,282 @@
+
+      cd iroha-ios/
+
+4.	Update dependencies:
+
+  .. code-block:: bash
+
+      carthage update --platform iOS
+
+5.	Go to sample project directory:
+
+  .. code-block:: bash
+
+      cd SwiftyIrohaExample
+
+6.	Update dependencies for the sample:
+
+  .. code-block:: bash
+
+      carthage update --platform iOS
+
+7.	Go to GRPC library source's location:
+
+  .. code-block:: bash
+
+      cd grpc-swift/
+
+8.	Remove old library sources:
+
+.. note:: Make sure You've located in ``grpc-swift/`` subfolder
+
+  .. code-block:: bash
+
+      # removes all files from the current directory
+      rm -rf ./*
+      #removes all hidden files too (so clean build can be done)
+      rm -rf ./.*
+ 
+9.	Download release version of GRCP from git to the current directory:
+
+  .. code-block:: bash
+
+      git clone --branch 0.3.3 https://github.com/grpc/grpc-swift.git .
+
+10.	Build library:
+
+  .. code-block:: bash
+
+      make
+
+11.	Go to the root of your playground folder (from the first step - path/to/your/folder/for/example/iroha-ios/project/):
+
+  .. code-block:: bash
+
+      cd ../../..
+
+.. note:: Make sure now You are located in ``path/to/your/folder/for/example/iroha-ios/project/`` folder
+
+12.	This step downloads script for client library which is needed to build client library. Clone it from the repository:
+
+  .. code-block:: bash
+
+      curl https://raw.githubusercontent.com/hyperledger/iroha/master/shared_model/packages/ios/ios-build.sh > ios-build.sh
+
+13.	Optional step. If You have issues with cloning during ios-build.sh execution do the following command before the script invocation:
+
+  .. code-block:: bash
+
+      sed -i '' 's|git://github.com/hyperledger/iroha-ed25519|https://github.com/hyperledger/iroha-ed25519.git|g' ios-build.sh
+
+14.	Make downloaded script executable:
+
+  .. code-block:: bash
+
+      chmod +x ios-build.sh
+
+15.	Finally, build the client iOS library with proper options - platform: OS | SIMULATOR | SIMULATOR64; build: Debug | Release :
+
+  .. code-block:: bash
+
+      ./ios-build.sh SIMULATOR64 Debug
+
+16.	The generated artifacts should be copied to the proper location (let's create it first):
+
+  .. code-block:: bash
+
+      # this command shows location for simulator artefacts
+      # use this command for device instead:
+      # mkdir -p iroha-ios/libs/iOS/
+      mkdir -p iroha-ios/libs/Simulator/
+ 
+17.	Copy generated binaries:
+
+  .. code-block:: bash
+
+      # this command shows location for simulator artefacts
+      # use this command for device instead:
+      # cp lib/* iroha-ios/libs/iOS/
+      cp lib/* iroha-ios/libs/Simulator/
+
+18.	Don't forget to copy generated headers:
+
+  .. code-block:: bash
+
+      cp -a include/. iroha-ios/headers/
+
+19.	Now it's time to manually config Xcode project for the sample application. Open SwiftyIroha.xcodeproj:
+
+.. image:: https://github.com/hyperledger/iroha/raw/develop/docs/image_assets/iroha_swift_guide/iroha_swift_guide_001.png
```

Comment:
I think it should be a local relative path, shouldn't it?

Reply:
Currently relative path is not supported, the same approach is done in another doc files

Reply:
For now on let's just leave it as it is. @neewy please confirm.

Reply:
Dear @l4l , @DenDoronin, @laSinteZ 

Due to the problem with https://github.com/soramitsu/localized_docs, and the fact that each project is in essence a collection of symbolic links pointing to rst files it is really hard to write their links so that they will point to the same directories in Iroha and localised_docs. 

Knowing that, we have a principle in https://github.com/hyperledger/iroha/blob/master/docs/README.md file, which is "you should reference images or assets using GitHub download link or an external service, since our localized docs can't use relative path to image assets or other resources"